python-sdk/docs/guides/databricks.rst

```diff
@@ -61,6 +61,35 @@ To use the ``aql.load_file()`` function, you will need to specify the path to th
         output_table=Table(conn_id="my_databricks_conn"),
     )
 
+To load data into databrick, you only need to set the ``AIRFLOW__ASTRO_SDK__DATABRICKS_CLUSTER_ID`` env variable
+so the Astro SDK knows where to send your load_file job.
+Autoloader Support
+==================
+
+Autoloader is a powerful data loading feature in Databricks that allows users to efficiently load large amounts of data into delta tables. It has several benefits over traditional methods such as COPY INTO:
+
+* Incremental loading: Autoloader can detect new files in a directory and only load those, rather than all files every time. This makes it more efficient for loading data on a regular basis.
+* Schema inference: Autoloader can automatically infer the schema of the data being loaded, making it easier to get up and running quickly with new datasets.
+* Scalability: Autoloader is designed to handle very large datasets, making it a more scalable option for data loading.
+
+By default, the Astro SDK uses autoloader to load data into Databricks. However, if you want to use COPY INTO instead, you can set the load_mode option in your load_options object like this:
+
+.. code-block:: python
+
+    from astro.databricks.load_options import DeltaLoadOptions
+    from astro.constants import DatabricksLoadMode
+
+    delta_options = DeltaLoadOptions.get_default_delta_options()
+    delta_options.load_mode = DatabricksLoadMode.COPY_INTO
+    aql.load_file(
+        input_file=File("data.csv"),
+        output_table=Table(conn_id="my_databricks_conn"),
+        load_options=delta_options,
+    )
```

Comment:
I would prefer having defaults set in the `__init__` so that we can do just this:
```suggestion
    aql.load_file(
        input_file=File("data.csv"),
        output_table=Table(conn_id="my_databricks_conn"),
        load_options=DeltaLoadOptions(load_mode=DatabricksLoadMode.COPY_INTO),
    )
```